src/test/scala/nl.knaw.dans.easy.deposit/MailerSpec.scala

```diff
@@ -50,6 +50,13 @@ class MailerSpec extends TestSupportFixture {
       .buildMessage(data, attachments, UUID.randomUUID(), msg4Datamanager = Option("")) shouldBe a[Success[_]]
   }
 
+  it should "not add empty attachments" in {
```

Comment:
I wonder how you can test this... It's not done below and I could not find a way (last week) to get a listing of email attachments from the mail object resultant of `.buildMessage`.

Reply:
whoops, I continued on #201